examples/text_to_image/train_text_to_image_xla.py

```diff
@@ -82,16 +81,13 @@ def start_training(self):
                 print(e)
                 break
             loss = self.step_fn(batch["pixel_values"], batch["input_ids"])
-            step_time = time.time() - last_time
-            if step >= 10:
-                times.append(step_time)
-            print(f"step: {step}, step_time: {step_time}")
-            if step%5 == 0:
-                print(f'step: {step}, loss: {loss}')
-            last_time = time.time()
+            
             self.global_step += 1
             step += 1
-        print(f"Average step time: {sum(times)/len(times)}")
+            if step == self.args.max_train_steps:
+                xm.wait_device_ops()
+                total_time = time.time() - last_time
```

Comment:
Should we add a break statement after this?

Reply:
Updated the condition to use for loop.